x-pack/legacy/plugins/infra/public/pages/metrics/index.tsx

```diff
@@ -112,26 +112,28 @@ export const MetricDetail = withMetricPageProviders(
                 })}
               />
               <DetailPageContent data-test-subj="infraMetricsPage">
-                <NodeDetailsPage
-                  name={name}
-                  requiredMetrics={filteredRequiredMetrics}
-                  sourceId={sourceId}
-                  timeRange={timeRange}
-                  parsedTimeRange={parsedTimeRange}
-                  nodeType={nodeType}
-                  nodeId={nodeId}
-                  cloudId={cloudId}
-                  metadataLoading={metadataLoading}
-                  isAutoReloading={isAutoReloading}
-                  refreshInterval={refreshInterval}
-                  sideNav={sideNav}
-                  metadata={metadata}
-                  addNavItem={addNavItem}
-                  setRefreshInterval={setRefreshInterval}
-                  setAutoReload={setAutoReload}
-                  triggerRefresh={triggerRefresh}
-                  setTimeRange={setTimeRange}
-                />
+                {metadata ? (
```

Comment:
:information_source: This was moved up from the `NodeDetailsPage` component to fix the conditional hook usage. There's probably a better way of handling this than not rendering anything, but that's what the component itself did (@simianhacker @phillipb).